update compose-preview/main failing with

> Task :cli:classes
> Task :cli:jar
> Task :cli:startScripts
> Task :cli:installDist
gradle/actions: Writing build results to /home/runner/work/_temp/.gradle-actions/build-results/__self_2-1779444868170.json

BUILD SUCCESSFUL in 1m 57s
115 actionable tasks: 111 executed, 4 from cache
Configuration cache entry stored.
Run echo "/opt/compose-preview/bin" >> "$GITHUB_PATH"
Run compose-preview show --json --timeout "$RENDER_TIMEOUT" > _previews.json
Found preview modules: samples:android, samples:android-alpha, samples:android-daemon-bench, samples:android-library, samples:android-screenshot-test, samples:cmp, samples:cmp-shared, samples:desktop-daemon-bench, samples:remotecompose, samples:sdk-matrix, samples:wear
FAILURE: Build failed with an exception.
* What went wrong:
> There were failing tests. See the report at: file:///home/runner/work/compose-ai-tools/compose-ai-tools/samples/sdk-matrix/build/reports/tests/composePreviewRender/index.html
* Try:
> Run with --scan to get full insights from a Build Scan (powered by Develocity).

Run with --verbose for full build output.

Failing tests in :samples:sdk-matrix:composePreviewRender (1):
  [RenderPreviewEntry(id=com.example.sdkmatrix.MatrixPreviewKt.SdkMatrixPreview_SdkMatrixPreview, functionName=SdkMatrixPreview, className=com.example.sdkmatrix.MatrixPreviewKt, sourceFile=src/main/kotlin/com/example/sdkmatrix/MatrixPreview.kt, params=RenderPreviewParams(name=SdkMatrixPreview, device=null, widthDp=280, heightDp=160, density=2.625, fontScale=1.0, showSystemUi=false, showBackground=false, backgroundColor=0, uiMode=0, locale=null, group=null, wrapperClassName=null, previewParameterProviderClassName=null, previewParameterLimit=2147483647, kind=COMPOSE), captures=[RenderPreviewCapture(advanceTimeMillis=null, scroll=null, animation=null, focus=null, focusGif=null, ambient=null, renderOutput=renders/com.example.sdkmatrix.MatrixPreviewKt.SdkMatrixPreview_SdkMatrixPreview.png, cost=1.0)], dataProducts=[])].classMethod
    Failed to create a Robolectric sandbox: Android SDK 36 requires Java 21 (have Java 17)
    java.lang.UnsupportedOperationException: Failed to create a Robolectric sandbox: Android SDK 36 requires Java 21 (have Java 17)
    at org.robolectric.plugins.DefaultSdkProvider$DefaultSdk.verifySupportedSdk(DefaultSdkProvider.java:170)
    at org.robolectric.RobolectricTestRunner.getSandbox(RobolectricTestRunner.java:262)
    at org.robolectric.RobolectricTestRunner.getSandbox(RobolectricTestRunner.java:64)
    at org.robolectric.internal.SandboxTestRunner.sandboxGroupingClassBlock(SandboxTestRunner.java:140)
    at org.robolectric.internal.SandboxTestRunner.classBlock(SandboxTestRunner.java:119)
Render failed
Error: Process completed with exit code 2.
Run actions/upload-artifact@043fb46d1a93c77aae656e7c1c64a875d1fc6a0a
With the provided path, there will be 199 files uploaded
Artifact name is valid!
Root directory input is valid!
Uploading artifact: composePreviewRender-reports.zip
Beginning upload of artifact content to blob storage
Uploaded bytes 338038
Finished uploading artifact content to blob storage!
SHA256 digest of uploaded artifact is a83e8bc1b6407b38ce5ffedafecc975ed8fde1bae5dbc5eaa218458de61d5bfd
Finalizing artifact upload
Artifact composePreviewRender-reports successfully finalized. Artifact ID 7158228065
Artifact composePreviewRender-reports has been successfully uploaded! Final size is 338038 bytes. Artifact ID is 7158228065
Artifact download URL: https://github.com/yschimke/compose-ai-tools/actions/runs/26281904762/artifacts/7158228065

--- a/samples/sdk-matrix/build.gradle.kts
+++ b/samples/sdk-matrix/build.gradle.kts
@@ -25,10 +25,15 @@ plugins {
   id("ee.schimke.composeai.preview")
 }
 
+// Defaults pinned to SDK 35 (not 36) so the no-override path renders cleanly under the project's
+// default JDK 17 toolchain — Robolectric refuses to bootstrap an SDK 36 sandbox without JDK 21+
+// (`DefaultSdkProvider.verifySupportedSdk`), which would fail every regular `preview-baselines`
+// run that doesn't set the matrix `-P` overrides. The nightly `sdk-matrix.yml` workflow always
+// passes explicit cell values, so it sweeps the full {35, 36, 37} range regardless of the default.
 val matrixCompileSdk: Int =
-  providers.gradleProperty("composeai.matrix.compileSdk").orNull?.toIntOrNull() ?: 36
+  providers.gradleProperty("composeai.matrix.compileSdk").orNull?.toIntOrNull() ?: 35
 val matrixTargetSdk: Int =
-  providers.gradleProperty("composeai.matrix.targetSdk").orNull?.toIntOrNull() ?: 36
+  providers.gradleProperty("composeai.matrix.targetSdk").orNull?.toIntOrNull() ?: 35
 val matrixMinSdk: Int =
   providers.gradleProperty("composeai.matrix.minSdk").orNull?.toIntOrNull() ?: 24
 val matrixSdkOverride: Int? =

